packages/camera/camera_android_camerax/pigeons/camerax_library.dart

```diff
@@ -46,6 +46,13 @@ class CameraPermissionsErrorData {
   String description;
 }
 
+abstract class InstanceManagerHostApi {
+  /// Clear the native `InstanceManager`.
+  ///
+  /// This is typically only used after a hot restart.
+  void clear();
+}
+
```

Comment:
@bparrishMines I'm using this PR to do some cleanup I need to do to match the internal plugin and use your updated instance manager. I was wondering where this is used? As the comment suggests, I figured this was used for hot restart, but I didn't see it used in your example.

Reply:
It's used in the initial creation of `BaseObject.globalInstanceManager`: https://github.com/bparrishMines/packages/blob/wrapper_example/packages/wrapper_example/lib/src/base_object.dart#L74.

This should translate to be used by your `JavaObject` class.

Reply:
I would also note that this also breaks some tests. So I had to mock the host api in the tests:

```dart

@GenerateMocks(<Type>[TestInstanceManagerHostApi])
void main() {
  TestWidgetsFlutterBinding.ensureInitialized();

  // Mocks the call to clear the native InstanceManager.
  TestInstanceManagerHostApi.setup(MockTestInstanceManagerHostApi());
...
```